flang/lib/Lower/ConvertExpr.cpp

```diff
@@ -2361,3 +2358,13 @@ fir::ExtendedValue Fortran::lower::createStringLiteral(
   return ScalarExprLowering{loc, converter, dummySymbolMap, dummyStmtCtx}
       .genStringLit(str, len);
 }
+
+fir::MutableBoxValue Fortran::lower::createSomeMutableBox(
+    mlir::Location loc, Fortran::lower::AbstractConverter &converter,
+    const Fortran::evaluate::Expr<Fortran::evaluate::SomeType> &expr,
+    Fortran::lower::SymMap &symMap) {
+  // Box address inquiry is not an array expression.
```

Comment:
It was intended to justify that we do not need to expose the `StatementContext` to the caller as in the other similar `createXX` because it makes no sens to treat `A+B` as a allocatable/pointer, `createSomeMutableBox` can only be called on pointer/allocatable variables.
I changed the comment to:

> MutableBox lowering StatementContext does not need to be propagated to the caller because the result value is a variable, not a temporary expression. The StatementContext clean-up can occur before using the resulting MutableBoxValue.

Reply:
Got it. Thanks.